contracts/zero-ex/contracts/src/ZeroEx.sol

```diff
@@ -34,11 +34,12 @@ contract ZeroEx {
 
     /// @dev Construct this contract and register the `Bootstrap` feature.
     ///      After constructing this contract, `bootstrap()` should be called
-    ///      to seed the initial feature set.
-    constructor() public {
+    ///      by `bootstrap()` to seed the initial feature set.
+    /// @param bootstrapper Who can call `bootstrap()`.
+    constructor(address bootstrapper) public {
         // Temporarily create and register the bootstrap feature.
         // It will deregister itself after `bootstrap()` has been called.
-        Bootstrap bootstrap = new Bootstrap(msg.sender);
+        Bootstrap bootstrap = new Bootstrap(bootstrapper);
         LibProxyStorage.getStorage().impls[bootstrap.bootstrap.selector] =
             address(bootstrap);
     }
```

Comment:
Previously it was the job of `InitialMigration` to deploy the `ZeroEx` contract. Now we deploy the `ZeroEx` contract (along with all the features) in a separate tx, but we still want `InitialMigration` to be the one to wire them all together. So we need to be able to decouple the deployer of `ZeroEx` from the bootstrapper (`InitialMigration`).

Reply:
Impetus for this change is that `FixinEIP712` wants the `ZeroEx` contract address to init its immutables, so we need* to have `ZeroEx` deployed beforehand.

*unless we want to predict the address via hash.

Reply:
No need to redeploy anything. Bootstrapper doesn't exist in the wild since it self-destructs.